src/rapidoc.js

```diff
@@ -566,6 +566,19 @@ export default class RapiDoc extends LitElement {
     }
   }
 
+  resetDoc() {
```

Comment:
`resetDoc` name is misleading, there are many states in a the doc, it is just reseting the loader and some classes.

IMO, we do not need a separate method for this, just put these statements in `loadSpec()` method